src/main/scala/chiseltest/stage/phases/Checks.scala

```diff
@@ -0,0 +1,35 @@
+package chiseltest.stage.phases
+
+import chiseltest.stage._
+import firrtl.AnnotationSeq
+import firrtl.options.{Dependency, OptionsException, Phase, TargetDirAnnotation}
+
+class Checks extends Phase {
+  override def prerequisites = Seq.empty
+
+  override def optionalPrerequisites = Seq(Dependency[Checks])
+
+  override def optionalPrerequisiteOf = Seq.empty
+
+  override def invalidates(a: Phase) = false
+
+  def transform(annotations: AnnotationSeq): AnnotationSeq = {
```

Comment:
yes, it was a little redundant, I also think I need to rethink about it.

Reply:
in that when the data is used and not present, it will cause a noisy error elsewhere?
If annotations needed are empty, it is a bug, like missing `TestNameAnnotation`, `TestFunctionAnnotation`
I think it's not redundant now.